akka-actor/src/main/scala/akka/util/JavaDurationConverters.scala

```diff
@@ -3,13 +3,13 @@
  */
 package akka.util
 import java.time.{ Duration ⇒ JDuration }
-import scala.concurrent.duration.Duration
+import scala.concurrent.duration.{ Duration, FiniteDuration }
 /**
  * INTERNAL API
  */
 private[akka] object JavaDurationConverters {
   final implicit class JavaDurationOps(val self: JDuration) extends AnyVal {
-    def asScala: Duration = Duration.fromNanos(self.toNanos)
+    def asScala: FiniteDuration = Duration.fromNanos(self.toNanos)
```

Comment:
One thing that could be nicer here would be to call `toCoarsest` to get one second in the Java duration end up as `1s` `FiniteDuration` (even if Java does not store the unit) rather than `1000000000 ns` when logged etc.

Reply:
`toCoarsest` returns a `Duration`, so maybe should deal with this in a separate issue or PR?

Reply:
Let's skip that for now then.